docs/code/mecsimcalc-library.md

```diff
@@ -0,0 +1,155 @@
+---
+sidebar_label: "mecsimcalc Library"
+sidebar_position: 4
+---
+
+# Mecsimcalc 0.0.2 documentation
+
+This library is designed to provide a set of functions for handling and converting various types of data, such as base64 encoded data, Pandas DataFrames, and Pillow images.
```

Comment:
In the description: can you add a bullet list with:
* The github repo saying that this is where the source code is
* The pypi page saying this is where you download it from